src/main/java/org/plumelib/util/StringsPlume.java

```diff
@@ -600,10 +605,11 @@ public static String removeWhitespaceAround(String arg, String delimiter) {
    * @param arg string to remove whitespace in
    * @param delimiter string to remove whitespace after
    * @return version of arg, with whitespace after delimiter removed
+   * @throws IllegalArgumentException if the second argument is an empty String
```

Comment:
Adding `@throws`  to the specification is usually the wrong thing to do.  It is better to state the requirements, and not specify what happens if they are violated.  You don't want clients to depend on the specific exception, and you don't want to restrict the flexibility to change the implementation in the future.

Here you are right that the specification is incomplete and should be improved -- thanks for noticing that.  However, a better fix is to change
```java
   * @param delimiter string to remove whitespace after
```
to
```java
   * @param delimiter a non-empty string to remove whitespace after
```

Reply:
I added @thows because i saw it a lot in documentation of other methods in this repo.

Reply:
In this file, there are only 5 `@throws`, and 4 were just added by you.  So I think there are not a lot of `@throws` in this file before your changes.

Reply:
I was referring to the whole repository. For example in Arrays Plume there are more than 15 @throws.

https://github.com/plume-lib/plume-util/blob/master/src/main/java/org/plumelib/util/ArraysPlume.java#L375